pom.xml

```diff
@@ -75,7 +75,7 @@ limitations under the License.
     <commons-dbcp2.version>2.5.0</commons-dbcp2.version>
     <commons-lang3.version>3.8</commons-lang3.version>
     <commons-pool2.version>2.6.0</commons-pool2.version>
-    <elasticsearch.version>6.2.4</elasticsearch.version>
+    <elasticsearch.version>7.0.0</elasticsearch.version>
```

Comment:
You didn't need to also change `org.codelibs.elasticsearch.module` dependency (see [integration in ES](https://discuss.elastic.co/t/5-0-0-using-painless-in-esintegtestcase/64447/13)) ? 

Is painless scripting working out of box ?

Reply:
Latest ES version is `7.0.1`

Reply:
I confirmed with codelibs owner that it works.

cc @marevol